### ISSUE TITLE
Sparky needs a logo

### ISSUE THREAD

**Issue by thibaultduponchelle** [melezhik/sparky#41]
This is my favorite issue :sunglasses:

**Comment by melezhik** [comment]
I like this one too 😄

**Comment by melezhik** [comment]
One day I created the one. Don't expect a lot 😄  - https://github.com/melezhik/Sparrow6/blob/master/logo.png , I even used it for a while, but still not sure is it going to be a Sparrow logo or Sparky logo. Or even throw it to trash bin 😄

**Comment by melezhik** [comment]
![logo](https://raw.githubusercontent.com/melezhik/Sparrow6/master/logo.png)

**Comment by thibaultduponchelle** [comment]
Wow, I really don’t like it ! Ahah 😁

**Comment by melezhik** [comment]
well, you can suggest your design as well  or maybe I'll drop new variants here ...

**Comment by thibaultduponchelle** [comment]
I will make a try, sorry for being harsh but It came from my heart 😄

**Comment by melezhik** [comment]
That's fine. I mean, like said  "Don't expect a lot". It's ok, people like different things ... If we have enough logos we can run a poll one day and include some people from raku community for the voting ... just a thoughts

**Comment by thibaultduponchelle** [comment]
First attempt: 

![sparky](https://user-images.githubusercontent.com/580360/103641534-2dd86100-4f52-11eb-9ab9-c560aa5c1a51.png)

**Comment by thibaultduponchelle** [comment]
Any update here ? 

Do you want me to try something else ? (color? animal?)

**Comment by melezhik** [comment]
Yeah, I would add more variants, if you don't mind. I like this dog's association btw :-) Initially because everything started with Sparrow I was thinking more in birdish way ... But dog is fine too.

**Comment by thibaultduponchelle** [comment]
Ah ah, I was writing [this](https://dev.to/thibaultduponchelle/what-if-i-tell-you-sparrow-3bed) before you wrote this comment

**Comment by melezhik** [comment]
Oh! I like your dev post on Sparrow, thank you very much for spreading the
word!




ср, 13 янв. 2021 г. в 09:13, thibaultduponchelle <notifications@github.com>:

**Comment by melezhik** [comment]
@thibaultduponchelle  do you mind if I share your dev.to post on raku reddit?

**Comment by thibaultduponchelle** [comment]
No problem of course, go ahead 😃

**Comment by thibaultduponchelle** [comment]
Here is a birdish attempt :
![sparkybird10](https://user-images.githubusercontent.com/580360/104595883-7fc86780-5673-11eb-8f27-536c97d97dde.png)

**Comment by thibaultduponchelle** [comment]
Or even without the checkboxes:
![sparkybird3](https://user-images.githubusercontent.com/580360/104570103-cdcc7380-5651-11eb-8cc2-e4e82a2423f8.png)

**Comment by melezhik** [comment]
@thibaultduponchelle hi! I am not sure if follow )), but if you are interested - here is my take - 

https://github.com/melezhik/sparky/blob/master/logos/sparky.png

![logo](https://github.com/melezhik/sparky/blob/master/logos/sparky.small.png?raw=true)
